# HG changeset patch
# User Rik <rik@octave.org>
# Date 1513033447 28800
#      Mon Dec 11 15:04:07 2017 -0800
# Branch stable
# Node ID f6016eeb27ec152951bac041ce2b5ad5c9fca107
# Parent  62add1b6910047e1f05ae3f3771a2fcdedad63ea
doc: Add seealso links between addlistener/dellistener.

* graphics.cc (Faddlistener, Fdellistener): Add seealso links between
these two functions.

diff --git a/libinterp/corefcn/graphics.cc b/libinterp/corefcn/graphics.cc
--- a/libinterp/corefcn/graphics.cc
+++ b/libinterp/corefcn/graphics.cc
@@ -11192,17 +11192,17 @@ Example:
 function my_listener (h, dummy, p1)
   fprintf ("my_listener called with p1=%s\n", p1);
 endfunction
 
 addlistener (gcf, "position", @{@@my_listener, "my string"@})
 @end group
 @end example
 
-@seealso{addproperty, hggroup}
+@seealso{dellistener, addproperty, hggroup}
 @end deftypefn */)
 {
   gh_manager::auto_lock guard;
 
   int nargin = args.length ();
 
   if (nargin < 3 || nargin > 4)
     print_usage ();
@@ -11251,16 +11251,17 @@ function my_listener (h, dummy, p1)
 endfunction
 
 c = @{@@my_listener, "my string"@};
 addlistener (gcf, "position", c);
 dellistener (gcf, "position", c);
 @end group
 @end example
 
+@seealso{addlistener}
 @end deftypefn */)
 {
   gh_manager::auto_lock guard;
 
   if (args.length () < 2 || args.length () > 3)
     print_usage ();
 
   double h = args(0).xdouble_value ("dellistener: invalid handle");
